src/main/java/ua/ithillel/lesson23/HeroService.java

```diff
@@ -16,16 +17,27 @@ public HeroService(HeroDao heroDao, HeroMovieService heroMovieService) {
     }
 
     public List<HeroDto> getHeroes() {
-        List<Hero> heroes = heroDao.findAll();
-        List<HeroDto> listDto = new ArrayList<>();
-        for (Hero hero : heroes) {
-            HeroDto heroDto = new HeroDto.HeroBuilder()
-                    .name(hero.getName())
-                    .build();
-            List<String> movies = heroMovieService.getPlayedIn(hero.getName());
-            listDto.add(heroDto);
-        }
-        return listDto;
+        return heroDao.findAll()
+                .stream()
+                .map(hero -> {
```

Comment:
```suggestion
                .map(hero -> new HeroDto.HeroBuilder()
                            .name(hero.getName())
                            .movies(heroMovieService.getPlayedIn(hero.getName())
                            .build()
```
1. в один return (тоді return не треба вказувати) 
2. movies не губляться